<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Double Contact</string>
  </property>
  <property name="windowIcon">
   <iconset resource="doublecontact.qrc">
    <normaloff>:/img/16x16/doublecontact_16x16.png</normaloff>:/img/16x16/doublecontact_16x16.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="wgtLeftTable">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lbLeft">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;Empty&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tvLeft">
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lbFilterLeft">
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leFilterLeft"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wgtRightTable">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lbRight">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;Empty&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tvRight">
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="lbFilterRight">
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leFilterRight"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Add (Ins)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEdit">
        <property name="text">
         <string>Edit (Enter)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemove">
        <property name="text">
         <string>Remove (Del)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSwapNames">
        <property name="text">
         <string>F2 Swap Names</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCompare">
        <property name="text">
         <string>F3 Compare</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSort">
        <property name="text">
         <string>F4 Sort</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnCopy">
        <property name="text">
         <string>F5 Copy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMove">
        <property name="text">
         <string>F6 Move</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>&amp;Open</string>
     </property>
     <addaction name="action_OpenFile"/>
     <addaction name="action_OpenDir"/>
    </widget>
    <widget class="QMenu" name="menuSaveAs">
     <property name="title">
      <string>S&amp;ave as</string>
     </property>
     <addaction name="action_SaveAsFile"/>
     <addaction name="action_SaveAsDir"/>
    </widget>
    <widget class="QMenu" name="menuRecent">
     <property name="title">
      <string>&amp;Recent</string>
     </property>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="action_Save"/>
    <addaction name="menuSaveAs"/>
    <addaction name="action_Close"/>
    <addaction name="menuRecent"/>
    <addaction name="actionRe_port"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Contact">
    <property name="title">
     <string>&amp;Contact</string>
    </property>
    <addaction name="action_Add"/>
    <addaction name="action_Edit"/>
    <addaction name="action_Remove"/>
    <addaction name="separator"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Move"/>
    <addaction name="separator"/>
    <addaction name="action_Join"/>
    <addaction name="actionSp_lit"/>
    <addaction name="separator"/>
    <addaction name="action_Swap_names"/>
    <addaction name="actionCompare_Result"/>
    <addaction name="actionS_plit_names"/>
    <addaction name="actionJoin_names"/>
    <addaction name="action_Drop_slashes"/>
    <addaction name="actionIntl_phone_prefix"/>
    <addaction name="actionFormat_phone_numbers"/>
    <addaction name="separator"/>
    <addaction name="action_Generate_full_name"/>
    <addaction name="actionParse_full_name"/>
    <addaction name="actionDrop_full_name"/>
    <addaction name="actionReverse_full_name"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_About_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Two_panels"/>
    <addaction name="action_Groups"/>
    <addaction name="separator"/>
    <addaction name="actionMessages"/>
    <addaction name="actionCalls"/>
    <addaction name="action_Files"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menu_List">
    <property name="title">
     <string>&amp;List</string>
    </property>
    <addaction name="actionCo_mpare"/>
    <addaction name="action_Sort"/>
    <addaction name="action_Hard_sort"/>
    <addaction name="action_Other_panel"/>
    <addaction name="action_Filter"/>
    <addaction name="actionS_wap_Panels"/>
    <addaction name="actionMass_tags_remove"/>
    <addaction name="actionS_tatistics"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_List"/>
   <addaction name="menu_Contact"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_OpenFile">
   <property name="text">
    <string>&amp;File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_OpenDir">
   <property name="text">
    <string>&amp;Directory</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_SaveAsFile">
   <property name="text">
    <string>&amp;File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_SaveAsDir">
   <property name="text">
    <string>&amp;Directory</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Two_panels">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Two panels</string>
   </property>
  </action>
  <action name="action_Options">
   <property name="text">
    <string>&amp;Options...</string>
   </property>
  </action>
  <action name="action_Add">
   <property name="text">
    <string>&amp;Add</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="action_Edit">
   <property name="text">
    <string>&amp;Edit</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="action_Remove">
   <property name="text">
    <string>&amp;Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_Move">
   <property name="text">
    <string>&amp;Move</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="action_Swap_names">
   <property name="text">
    <string>&amp;Swap names</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionCo_mpare">
   <property name="text">
    <string>&amp;Compare/Uncompare</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="action_Sort">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Sort</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="action_Other_panel">
   <property name="text">
    <string>&amp;Other panel</string>
   </property>
   <property name="shortcut">
    <string>Tab</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Filter">
   <property name="text">
    <string>&amp;Filter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionCompare_Result">
   <property name="text">
    <string>C&amp;ompare Result</string>
   </property>
   <property name="shortcut">
    <string>Shift+F3</string>
   </property>
  </action>
  <action name="actionS_plit_names">
   <property name="text">
    <string>S&amp;plit names</string>
   </property>
   <property name="shortcut">
    <string>Shift+F2</string>
   </property>
  </action>
  <action name="action_Drop_slashes">
   <property name="text">
    <string>&amp;Drop slashes</string>
   </property>
  </action>
  <action name="action_Generate_full_name">
   <property name="text">
    <string>&amp;Generate full name</string>
   </property>
  </action>
  <action name="action_Join">
   <property name="text">
    <string>&amp;Join</string>
   </property>
  </action>
  <action name="actionDrop_full_name">
   <property name="text">
    <string>Drop &amp;full name</string>
   </property>
  </action>
  <action name="actionSp_lit">
   <property name="text">
    <string>Sp&amp;lit</string>
   </property>
  </action>
  <action name="actionIntl_phone_prefix">
   <property name="text">
    <string>&amp;Internationalize phone prefix</string>
   </property>
  </action>
  <action name="actionS_wap_Panels">
   <property name="text">
    <string>S&amp;wap Panels</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionReverse_full_name">
   <property name="text">
    <string>Re&amp;verse full name</string>
   </property>
  </action>
  <action name="actionS_tatistics">
   <property name="text">
    <string>S&amp;tatistics</string>
   </property>
  </action>
  <action name="actionRe_port">
   <property name="text">
    <string>Re&amp;port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Hard_sort">
   <property name="text">
    <string>&amp;Hard sort</string>
   </property>
  </action>
  <action name="action_Groups">
   <property name="text">
    <string>&amp;Groups...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_About_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionFormat_phone_numbers">
   <property name="text">
    <string>Format phone &amp;numbers</string>
   </property>
  </action>
  <action name="actionSeparator1">
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionSeparator2">
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionSeparator3">
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionSeparator4">
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionCopy_text">
   <property name="text">
    <string>Copy text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionJoin_names">
   <property name="text">
    <string>Join names</string>
   </property>
  </action>
  <action name="actionParse_full_name">
   <property name="text">
    <string>Parse full name</string>
   </property>
  </action>
  <action name="actionMessages">
   <property name="text">
    <string>&amp;Messages...</string>
   </property>
   <property name="shortcut">
    <string>Shift+F7</string>
   </property>
  </action>
  <action name="actionCalls">
   <property name="text">
    <string>&amp;Calls...</string>
   </property>
   <property name="shortcut">
    <string>Shift+F8</string>
   </property>
  </action>
  <action name="actionMass_tags_remove">
   <property name="text">
    <string>Mass tags remove</string>
   </property>
  </action>
  <action name="action_Files">
   <property name="text">
    <string>&amp;Files...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="doublecontact.qrc"/>
 </resources>
 <connections/>
</ui>
